scripts/build-vue3.sh

```diff
@@ -1,6 +1,13 @@
 #!/bin/sh
 set -e
 
-vue-demi-switch 3
+# use v3 for src/util/vue
```

Comment:
read existing content, and save that in a variable, then set it back (avoids a bug or accidentally having a diff if the original changes?)